graphqlhub-schemas/src/facebook.js

```diff
@@ -0,0 +1,160 @@
+// https://developers.facebook.com/tools/explorer
+
+import {
+  getId as getNode
+} from './apis/facebook';
+
+import {
+  graphql,
+  GraphQLSchema,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLNonNull,
+  GraphQLInt,
+  GraphQLEnumType,
+  GraphQLBoolean,
+  GraphQLList,
+  GraphQLInterfaceType,
+  GraphQLID
+} from 'graphql';
+
+import {
+  nodeDefinitions,
+  fromGlobalId,
+
+  connectionDefinitions,
+  connectionArgs,
+  connectionFromArray,
+} from 'graphql-relay';
+
+const getTokenFromAST = (ast) => {
```

Comment:
@leebyron any advice or examples on a better way to do this? I want to pass in the API token once at some top-level field, and then ensure all subsequent API calls can access it